src/main/resources/documents_composition_templates/NotificationAARForEMAIL.html

```diff
@@ -216,7 +217,7 @@ <h1 style="font-size:32px; line-height:40px;">Hai ricevuto una notifica</h1>
                                                         <p>Hai ricevuto su SEND - Servizio Notifiche Digitali una notifica da parte di <strong>${notification.sender.paDenomination}</strong> con Codice IUN <strong>${notification.iun}</strong>. Una notifica è una <strong>comunicazione a valore legale</strong> composta da una avviso di avvenuta ricezione e da uno o più documenti.</p>
                                                         <!-- Perfezionamento Notifica -->
                                                         <p>Hai fino a 120 giorni dalla <a href="${PNFaqCompletionMomentURL}" class="link" style="color: #0073E6">data di perfezionamento</a> della notifica per <strong>visualizzare i documenti</strong>.</p>
-                                                        <a title="Visualizza documenti" class="button" href="${piattaformaNotificheURL}" style="color: #FFFFFF;">Visualizza documenti</a>
```

Comment:
Even if added to the class button, I would keep both, the inline style is required for text in outlook 2013 and prior versions